anomaly_detector/config.py

```diff
@@ -51,6 +51,8 @@ class Configuration():
     MODEL_PATH = ""
     W2V_MODEL_PATH_CALLABLE = join_w2v_model_path
     W2V_MODEL_PATH = ""
+    MODEL_BACKUP_SYSTEM= ""
```

Comment:
small nitpick: I think `BACKUP` is misleading, we dont do a backup, but your storing the model there.
So to be consistent with `--modelstore`, let's rename to `MODEL_STORE`.
The arg should be `--model-store`.
And the `S3_MODEL_UPLOAD_PATH` can be just `MODEL_STORE_PATH`.

Reply:
Just pushed up an update. So this should be resolved?